CMakeLists.txt

```diff
@@ -62,45 +43,5 @@ include_directories(
 add_definitions(${OPTSCHED_EXTRA_DEFINITIONS})
 link_directories(${OPTSCHED_EXTRA_LINK_LIBRARIES})
 
-if(LLVM_VERSION VERSION_LESS 7.0)
-  add_definitions(-DLLVM_DEBUG=DEBUG)
-endif()
-
-if(NOT llvm_subproject)
-  include(GetLocalLLVM)
-
-  set(args)
-  if(OPTSCHED_INCLUDE_TESTS)
-    list(APPEND args GTEST)
-  endif()
-
-  get_local_llvm(${args})
-endif()
```

Comment:
Don't delete this. It makes it possible to build and test (unit test) OptSched on its own without needing to build all of LLVM. This was a lifesaver when I wanted to build OptSched in many different branches, while all linking to the same LLVM version.

Reply:
Thanks for pointing that out.